src/WebJobs.Script/Description/PowerShell/PowerShellFunctionInvoker.cs

```diff
@@ -261,18 +261,33 @@ internal static List<string> GetModuleFilePaths(string rootScriptPath, string fu
         {
             List<string> modulePaths = new List<string>();
             string functionFolder = Path.Combine(rootScriptPath, functionName);
+            string rootModuleDirectory = Path.Combine(rootScriptPath, PowerShellConstants.ModulesFolderName);
             string moduleDirectory = Path.Combine(functionFolder, PowerShellConstants.ModulesFolderName);
-            if (Directory.Exists(moduleDirectory))
+            string[] searchDirectories = new string[] { rootModuleDirectory, moduleDirectory };
+            modulePaths.AddRange(AddToModulePaths(searchDirectories));
+            return modulePaths;
+        }
+
+        internal static List<string> AddToModulePaths(string[] directories)
```

Comment:
Overall, I would say that you should add tests for this and ensure that loading of modules works without errors when loading from both **rootModuleDirectory** and **moduleDirectory**.  Verify that if a DLL is duplicated in both directories, the load behavior is as expected.